playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/Logging.java

```diff
@@ -0,0 +1,35 @@
+package com.google.androidbrowserhelper.playbilling.provider;
+
+import android.util.Log;
+
+import com.android.billingclient.api.BillingClient;
+import com.android.billingclient.api.BillingResult;
+
+/**
+ * Consolidates all the logging for this package in one place.
+ */
+public class Logging {
+    private static final String TAG = "TwaBilling";
```

Comment:
I wonder if it's worth differentiating from the other logging? Eg: `TwaBilling.digitalgoods` and `TwaBilling.provider`.

Reply:
I'd like to just do `adb logcat | grep TwaBilling` and get everything. But yeah, I like your idea, I don't want the tags to get too long so I'll do it as `TwaBilling.P` and `TwaBilling.DG`.